docs/reference/commandline/secret_create.md

```diff
@@ -28,7 +28,7 @@ Options:
 ## Description
 
 Creates a secret using standard input or from a file for the secret content. You must run this
-command on a manager node.
+command on a manager node. We can get details about created secret from [here](https://docs.docker.com/engine/swarm/secrets/#about-secrets).
```

Comment:
I was thinking something like this, would that work for you?

```Markdown
Creates a secret using standard input or from a file for the secret content. You must run this
command on a manager node.

For detailed information about using secrets, refer to [manage sensitive data with Docker secrets](https://docs.docker.com/engine/swarm/secrets/).
```

Perhaps the same should be added to the other "secret" pages in the command-line reference.